.azure-pipelines/pr_test_scripts.yaml

```diff
@@ -333,6 +332,12 @@ onboarding_t0:
   - test_pktgen.py
   - arp/test_unknown_mac.py
   - hash/test_generic_hash.py
+  - dualtor/test_orchagent_active_tor_downstream.py
+  - dualtor/test_orchagent_mac_move.py
+  - dualtor/test_orchagent_standby_tor_downstream.py
+  - dualtor/test_standby_tor_upstream_mux_toggle.py
```

Comment:
need to sync with latest master, these dualtor scripts were put in to t0 job

Reply:
> need to sync with latest master, these dualtor scripts were put in to t0 job

Sure